src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+
+        for (int i = 0; i < 3; i++) {
+            balls[i] = new Lottery().getRandomBall();
+            System.out.println(balls[i].toString());
```

Comment:
```suggestion
            System.out.println(balls[i]);
```

we dont need to call toString method here, it will be called "under the hood"